shopfloor_reception/tests/reception_return_common.py

```diff
@@ -86,12 +86,13 @@ def deliver(cls, pickings):
             if not ready_picking:
                 break
             for line in ready_picking.move_line_ids:
-                line.qty_done = line.reserved_uom_qty
+                line.qty_picked = line.quantity
+                line.picked = True
             ready_picking._action_done()
 
     @classmethod
     def partial_deliver(cls, picking, qty_done):
-        picking.move_line_ids.write({"qty_done": qty_done})
+        picking.move_line_ids.write({"qty_picked": qty_done, "picked": True})
```

Comment:
```suggestion
       picking.move_line_ids.write({"qty_picked": qty_done, "picked": True})
```
the inverse will set `picked`